test/unit/lazyAndCommands/lazy-devices-test.js

```diff
@@ -249,7 +249,7 @@ describe('IoT Agent Lazy Devices', function() {
                 },
                 headers: {
                     'fiware-service': 'smartGondor',
-                    'fiare-servicepath': 'gardens'
+                    'fiware-servicepath': 'gardens'
```

Comment:
Seems to be "stetical" from the point of view of the tests, but better to have it with the right name.

NTC (just informative)